.github/workflows/build.yaml

```diff
@@ -31,6 +29,10 @@ jobs:
         run: |-
           cargo install cargo-make
 
+      - name: Installing protofetch
+        run: |-
+          cargo install --git https://github.com/knox-networks/protofetch --branch tracing
```

Comment:
Reposting:
we should stash a binary in the releases tag
so we can `wget` instead of spending a while building from source